src/mutations/member.test.ts

```diff
@@ -205,6 +205,44 @@ describe('Member Mutations', () => {
       expect(newData).toMatchObject(response);
     });
 
+    it(`Successfully edit member's username and ensures trimming id = ${member.id}`, async () => {
+      const response = { ...member, name: newMember.name };
+      // set random data in cache
+      queryClient.setQueryData(memberKeys.current().content, member);
+      const endpoints = [
+        {
+          response,
+          method: HttpMethod.Patch,
+          route,
+        },
+      ];
+      const patchSpy = vi.spyOn(axios, 'patch');
+      const mockedMutation = await mockMutation({
+        mutation,
+        wrapper,
+        endpoints,
+      });
+      const newUsernameWithTrailingSpace = 'newname  ';
+
+      await act(async () => {
+        mockedMutation.mutate({
+          id: member.id,
+          name: newUsernameWithTrailingSpace.trim(),
```

Comment:
Here you should not trim the input. You want to test that if you provide an input that has white space at the end, it will be removed by the hook. If you remove it before giving it to the hook you will have tested nothing.

Reply:
@spaenleh By saying the hook, you mean useEditMember?

Reply:
@mariembencheikh Yes, I mean the mutation hook useEditMember. 
Basically what you do here is that you:
1. create a newUsername = `'test '`
2. trim that username so it becomes `'test'`
3. give the trimmed username to the mutation
So the mutations actually never does any trimming. This currently does not test that trimming is performed in the mutation.